system-parachains/coretime/coretime-kusama/src/coretime.rs

```diff
@@ -123,13 +167,41 @@ impl CoretimeInterface for CoretimeAllocator {
 
 	fn request_revenue_info_at(when: RCBlockNumberOf<Self>) {
 		use crate::coretime::CoretimeProviderCalls::RequestRevenueInfoAt;
-		let _request_revenue_info_at_call =
+		let request_revenue_info_at_call =
 			RelayRuntimePallets::Coretime(RequestRevenueInfoAt(when));
 
-		log::debug!(
-			target: "runtime::coretime",
-			"`request_revenue` is unmiplemented on the relay."
-		);
+		// Weight for `request_revenue_at` from Kusama runtime benchmarks:
+		// `ref_time` = 37_637_000 + (3 * 25000000) + (6 * 100000000) = 712637000
+		// `proof_size` = 6428
+		// Add 5% to each component and round to 2 significant figures.
+		//
+		// This benchmark has been transplanted from a testnet and not rerun, so adding a healthy
+		// buffer. TODO refine when benchmarks are run.
+		let call_weight = Weight::from_parts(1_000_000_000, 20_000);
```

Comment:
may be we should move weights to a separate sub crate, so we can import them with low cost and use it here.
until then we really need a test for this, it easy to break it.

Reply:
We have here all the runtimes. Writing a test should be quite easy.

Reply:
https://github.com/polkadot-fellows/runtimes/issues/403